.github/actions/setup-nvidia/action.yml

```diff
@@ -16,6 +20,11 @@ runs:
       uses: nick-fields/retry@3e91a01664abd3c5cd539100d10d33b9c5b68482
       env:
         DRIVER_VERSION: ${{ inputs.driver-version }}
+        GPU_ARCH_TYPE: ${{ inputs.GPU_ARCH_TYPE }}
+        ${{ if eq(inputs.GPU_ARCH_TYPE , 'cuda-aarch64') }}:
+            ARCH_TYPE: sbsa
+        ${{ else }}:
```

Comment:
this else would be x86_64 by default, so we can remove this if you set it to default.